### PR TITLE
Eliminate include validation

### LINKED ISSUE #33: Eliminate includes validation
The concept of include files in the `/includes` directory is being eliminated.  For the validator, we should:

* Remove validation of these files against the `/includes` metaschema
* Remove references to the `/includes` directory in comments and tests for clarity

### PR DESCRIPTION
**Pull request by alanisaac**
Fixes #33

### CODE REVIEW

File: ocsf_validator/type_mapping.py [hunk 1]
```diff
@@ -9,8 +9,6 @@
     VersionMatcher(),
     DictionaryMatcher(),
     CategoriesMatcher(),
-    IncludeMatcher(),
-    CategoriesMatcher(),
```
Comment (alanisaac) [inline ocsf_validator/type_mapping.py:13]: `note`: unrelated, but I noticed `CategoriesMatcher` appeared twice in this list.

**Review by query-jeremy — DISMISSED** [review]
LGTM. Thanks, Alan!

**Comment by query-jeremy** [comment]
Can you bump the version number in `pyproject.toml`?

**Comment by alanisaac** [comment]
> Can you bump the version number in `pyproject.toml`?

Oh -- yep!  fixed in [b2cf4f5](https://github.com/ocsf/ocsf-validator/pull/32/commits/b2cf4f5a842ef0704c29eaf56b636b73000b5cb1)

**Review by floydtree — APPROVED** [review]

**Review by query-jeremy — APPROVED** [review]

**Pull request merged**